index.html

```diff
@@ -278,7 +282,9 @@
 
       <input class="muliFont" type="button" id="change-prop" value="Submit">
       <br><br>
-      <select id="select-prop" class="muliFont"><option selected>Choose a property</option></select>
```

Comment:
Not sure, there seems to be no difference between the code though. I think Ashley might have modified this.